src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/ApprenticeController.cs

```diff
@@ -366,12 +367,55 @@ public IActionResult ConfirmRestart(DatalockConfirmRestartRequest request)
         [HttpPost]        
         [Route("{apprenticeshipHashedId}/datalock/confirmrestart", Name = RouteNames.ConfirmRestart)]
         [Authorize(Policy = nameof(PolicyNames.HasAccountOwnerPermission))]
-        public async Task<IActionResult> ConfirmRestart(DatalockConfirmRestartViewModel viewModel)
+        public async Task<ActionResult> ConfirmRestart(DatalockConfirmRestartViewModel viewModel)
         {            
             if (viewModel.SendRequestToEmployer.HasValue && viewModel.SendRequestToEmployer.Value)
             {
                 var request = await _modelMapper.Map<TriageDataLocksRequest>(viewModel);
-                await _commitmentsApiClient.TriageDataLocks(viewModel.ApprenticeshipId, request);                
+                await _commitmentsApiClient.TriageDataLocks(viewModel.ApprenticeshipId, request);
+            }
+
+            return RedirectToAction("Details", "Apprentice", new { viewModel.ProviderId, viewModel.ApprenticeshipHashedId });
+        }
+
+        [HttpGet]
+        [Route("{apprenticeshipHashedId}/datalock", Name = RouteNames.UpdateDateLock)]
```

Comment:
`[DasAuthorize(CommitmentOperation.AccessApprenticeship)]` required here and on other Get actions with {apprenticeshipHashedId} in the route.